catalogs/catalog-kafka/src/main/java/org/apache/gravitino/catalog/kafka/KafkaCatalogPropertiesMetadata.java

```diff
@@ -30,13 +30,17 @@ public class KafkaCatalogPropertiesMetadata extends BaseCatalogPropertiesMetadat
   public static final String BOOTSTRAP_SERVERS = "bootstrap.servers";
 
   private static final Map<String, PropertyEntry<?>> KAFKA_CATALOG_PROPERTY_ENTRIES =
-      Collections.singletonMap(
-          BOOTSTRAP_SERVERS,
-          PropertyEntry.stringRequiredPropertyEntry(
+      ImmutableMap.<String, PropertyEntry<?>>builder()
+          .put(
               BOOTSTRAP_SERVERS,
-              "The Kafka broker(s) to connect to, allowing for multiple brokers by comma-separating them",
-              false /* immutable */,
-              false /* hidden */));
+              PropertyEntry.stringRequiredPropertyEntry(
+                  BOOTSTRAP_SERVERS,
+                  "The Kafka broker(s) to connect to, allowing for multiple brokers by "
+                      + "comma-separating them",
+                  false /* immutable */,
+                  false /* hidden */))
+          .putAll(BASIC_CATALOG_PROPERTY_ENTRIES)
+          .build();
```

Comment:
@mchades can you please check this part?

Reply:
This part LGTM